app/server/lib/OIDCConfig.ts

```diff
@@ -217,7 +217,9 @@ export class OIDCConfig {
   private _makeUserProfileFromUserInfo(userInfo: UserinfoResponse): Partial<UserProfile> {
     return {
       email: String(userInfo[ this._emailPropertyKey ]),
-      name: this._extractName(userInfo)
+      name: this._extractName(userInfo),
+      connectId: userInfo.sub,
+      loginMethod: 'External',
```

Comment:
I assumed the `loginMethod: 'External'` is meant to be used for such case.

I can also add this property for `UserProfile`s created SAMLConfig.ts if that's relevant.